src/braket/circuits/observables.py

```diff
@@ -529,12 +526,13 @@ class Hermitian(Observable):
     """Hermitian matrix as an observable."""
 
     # Cache of eigenpairs
-    _eigenpairs = {}
+    _eigenpairs: ClassVar = {}
```

Comment:
Why the need for the type annotation?

Reply:
https://docs.astral.sh/ruff/rules/mutable-class-default/ - as it is mutable, we should label it as such.